### PR TITLE
Update partner.txt

### PR DESCRIPTION
**Pull request by bnomei**
added some more expertises based on recent projects

### CODE REVIEW

File: content/partners/0_bruno-meilick/partner.txt [hunk 1]
```diff
@@ -92,6 +92,9 @@ Expertise:
 - Relational Databases and MongoDB
 - PHP, Alpine.js, Tailwind CSS
 - CI/CD
+- Setups with 2000+ content pages
+- Setups with 500+ user accounts
+- Multi-developer support possible
```
Comment (lukasbestle) [inline content/partners/0_bruno-meilick/partner.txt]: What do you mean with "multi-developer support"? As a reader I honestly wouldn't know.

Comment (bnomei) [reply content/partners/0_bruno-meilick/partner.txt]: Yeah I struggled with the naming there. 

It's if a client decides to hire me I can provide a direct fallback developer as a contact. that kind thing is very often requested for 10k+ or long-term projects.

i am open to suggestions on how to make that better to understand.

Comment (lukasbestle) [reply content/partners/0_bruno-meilick/partner.txt]: Maybe "Access to my developer network for critical projects"? But happy to merge your PR as is if you prefer. Indeed not easy to find a good wording.

Comment (bnomei) [reply content/partners/0_bruno-meilick/partner.txt]: lets just cut the last item

**Pull request merged**